From 02be30908f6b8424e0ae0f4e64fb892912d252d1 Mon Sep 17 00:00:00 2001
From: Harald Sitter <sitter@kde.org>
Date: Tue, 2 Dec 2014 13:29:52 +0100
Subject: [PATCH 15/16] fix library name in prf file to use the lib name
 variable

the lib name might be suffixed

(reviewed as part of r121323)
---
 crypto.prf.cmake | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/crypto.prf.cmake b/crypto.prf.cmake
index c966b23..f25440f 100644
--- a/crypto.prf.cmake
+++ b/crypto.prf.cmake
@@ -16,10 +16,10 @@ exists($$QCA_LIBDIR/qca.framework) {
 isEmpty(LINKAGE) {
 	INCLUDEPATH += $$QCA_INCDIR/QtCrypto
 	LIBS += -L$$QCA_LIBDIR
-	LINKAGE = -lqca
+	LINKAGE = -l@QCA_LIB_NAME@
 	CONFIG(debug, debug|release) {
-		windows:LINKAGE = -lqcad
-		mac:LINKAGE = -lqca_debug
+		windows:LINKAGE = -l@QCA_LIB_NAME@d
+		mac:LINKAGE = -l@QCA_LIB_NAME@_debug
 	}
 }
 
-- 
1.9.3

